src/modules/flyout/flyout-adapter.service.ts

```diff
@@ -1,61 +1,40 @@
 import {
-  ApplicationRef,
-  ComponentFactoryResolver,
-  ComponentRef,
-  EmbeddedViewRef,
+  ElementRef,
   Injectable,
-  Injector,
   Renderer2,
   RendererFactory2
 } from '@angular/core';
 
-import { SkyFlyoutComponent } from './index';
 import { SkyWindowRefService } from '../window';
 
 @Injectable()
 export class SkyFlyoutAdapterService {
-
   private renderer: Renderer2;
 
   constructor(
-    private appRef: ApplicationRef,
-    private injector: Injector,
     private rendererFactory: RendererFactory2,
-    private resolver: ComponentFactoryResolver,
     private windowRef: SkyWindowRefService
   ) {
-      this.renderer = this.rendererFactory.createRenderer(this.windowRef.getWindow().document.body, undefined);
+    this.renderer = this.rendererFactory.createRenderer(undefined, undefined);
```

Comment:
Oops. Good catch there.... I totally didn't mean to check that in...... was testing something and missed removing it.